### PR TITLE
Expose enableDrag property for showBottomSheet

### LINKED ISSUE #90664: [material] Expose `enableDrag` property for showBottomSheet
Hi,

I need a BottomSheet that users can‘t draging to close，but while calling `showBottomSheet` there is no option to disable the enableDrag property. Which is not exposed to method `showBottomSheet` in bottom_sheet.dart.

The `showModalBottomSheet` method has enableDrag property but doesn't fit our needs, we need to interact with  the rest of the app.

So I'd suggest exposing the enableDrag property to the `showBottomSheet` method, like the `showModalBottomSheet` method.

### PR DESCRIPTION
**Pull request by sanlittt**
fixed #90664 

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

### CODE REVIEW

**Comment by sanlittt** [comment]
@googlebot I signed it!

**Comment by sanlittt** [comment]
@googlebot I fixed it.

**Comment by gspencergoog** [comment]
Hi @sanlittt, thanks for the PR!

Looks like there are some analysis errors and test failures that need to be fixed, but this looks like a reasonable change to me.  Thanks for writing some good tests!

Please correct the failures, and I'll be happy to give it a final review.

File: packages/flutter/lib/src/material/scaffold.dart [hunk 1]
```diff
@@ -2363,7 +2364,7 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin, Resto
     bottomSheet = _StandardBottomSheet(
       key: bottomSheetKey,
       animationController: animationController,
-      enableDrag: !isPersistent,
+      enableDrag: enableDrag ?? (!isPersistent),
```
Comment (gspencergoog) [inline packages/flutter/lib/src/material/scaffold.dart:2367]: The extra parenthesis are not needed.

**Review by gspencergoog — APPROVED** [review]
32384589-a60f0e74-c078-11e7-9bc1-e5b5287aea9d

Remove the extra parenthesis, and this is good to go.

**Comment by skia-gold** [comment]
Gold has detected about 1 new digest(s) on patchset 9.
View them at https://flutter-gold.skia.org/cl/github/90932

**Comment by skia-gold** [comment]
Gold has detected about 1 new digest(s) on patchset 10.
View them at https://flutter-gold.skia.org/cl/github/90932

**Review by Piinks — CHANGES_REQUESTED** [review]
Hi @sanlittt welcome! 🎉 
For the public `showBottomSheet` methods where you have added `enableDrag`, can you add some documentation for what `enableDrag` means and what the default behavior is?

**Comment by sanlittt** [comment]
> Hi @sanlittt welcome! 🎉 For the public `showBottomSheet` methods where you have added `enableDrag`, can you add some documentation for what `enableDrag` means and what the default behavior is?

done

**Review by Piinks — APPROVED** [review]
LGTM! Thank you for the contribution.

**Pull request merged**